### DISCUSSION TITLE
Create post-fit plots with post-fit results

### DISCUSSION THREAD

**Discussion by Tomoya-Iizawa** [scikit-hep/cabinetry#456] [Q&A]
Hi,

Is there any way to create post-fit distribution with using post-fit results from other distribution?
For example, obtainining fit results from the fit for mll distribution, and create pT distribution applying the fit results from the mll distribution?

**Comment by alexander-held** [comment]
Yes, this is one of the reasons why [`model_utils.match_fit_results`](https://cabinetry.readthedocs.io/en/latest/api.html#cabinetry.model_utils.match_fit_results) was added. Create a workspace with the distributions you want to fit, perform the fit, obtain the fit results object. Then create a new workspace with the other distribution(s) for which you want to create the post-fit plots, use the `model_utils.match_fit_results` function to translate your existing fit results object to one suitable for that new model containing the new distributions, and use that translated fit results object to obtain post-fit plots for your new distributions. Please have a look at #431 for some more details.
